packages/utils/popup-manager.ts

```diff
@@ -44,10 +47,9 @@ const onModalClick = () => {
 }
 
 let hasModal = false
-let zIndex: number
 
 const getModal = function (): HTMLElement {
-  if (!isClient) return
+  if (!isClient) return undefined as any
```

Comment:
Should we mark the return type as `HTMLElement | undefined` instead of this?

Reply:
In the case of SSR, `getModal` should not be called, so `undefined` will not happen.